Biblioteka_Zadanie9.4/models.py

```diff
@@ -0,0 +1,52 @@
+import json
+
+
+class Library:
+    def __init__(self):
+        try:
+            with open("booklibrary.json", "r") as f:
+                items = json.load(f)
+                sorted_items = sorted(items, key=lambda k: k["price"], reverse=True)
+                self.allitems = sorted_items
+        except FileNotFoundError:
+            self.allitems = []
+
+    def all(self):
+        with open("booklibrary.json", "r") as f:
+            items = json.load(f)
+            self.allitems = sorted(items, key=lambda k: k["price"], reverse=True)
+        return self.allitems
+
+    def get(self, id):
+        book = [book for book in self.all() if book["id"] == id]
```

Comment:
za każdym razem jak chcesz odnaleźć w pliku jedną książkę to musisz załadować cały plik do pamięci RAM, a potem przeiterować przez wszystkie książki żeby znaleźć tylko jedną. Jest dużo rzeczy, które możemy zrobić żeby to naprawić:

1. spróbujmy trzymać w pamięci już raz załadowane dane z pliku.
2. spróbujmy użyć słownika zamiast listy. wtedy nie będziemy musieli wyszukiwać przez całą listę tylko wyszukamy sobie po identyfikatorze książki. Słownik by miał klucze odpowiadające id książki a wartości to obiekty tych ksiażek